Flow.Launcher/MainWindow.xaml

```diff
@@ -92,7 +93,7 @@
                             </ContextMenu>
                         </TextBox.ContextMenu>
                     </TextBox>
-                    <Image Source="{Binding Image, IsAsync=True}" Width="48"  HorizontalAlignment="Right" />
+                    <svgc:SvgControl Source="{Binding Image, Mode=OneWay}" HorizontalAlignment="Right" Width="48" Height="48"/>
```

Comment:
same how come this is one way now?

Reply:
This one is oneway because the search icon won't change.

Reply:
hmm i think if were are to implement something like change icon based on result selection or plugin installed indication in other issues, this might be good to remain async and two way, what do you think?

Reply:
> > now that we switched to svg, should we archive mainsearch.png into the 'doc' folder only for future reference instead?
> 
> If we do deside to switch to svg, we can archive the png one, since it is only used here.

haha well, i think we should, image scaling is better from svg.

Reply:
> hmm i think if were are to implement something like change icon based on result selection or plugin installed indication from other issues, this might be good to remain async and two way, what do you think?

hmmmm seems you are right, I will change it.

Reply:
Actually, I don't believe we need async. Async may cause a small period of nothing, like the image flickering. Even if we changes the icon, it should be fast because no need to extract thumbnail from files.

Reply:
ok cool no probs